apps/prairielearn/src/server.js

```diff
@@ -71,7 +70,11 @@ import { markAllWorkspaceHostsUnhealthy } from './lib/workspaceHost';
 import { createSessionMiddleware } from '@prairielearn/session';
 import { PostgresSessionStore } from './lib/session-store';
 import { pullAndUpdateCourse } from './lib/course';
-import { selectJobsByJobSequenceId } from './lib/server-jobs';
+import {
+  init as serverJobsInit,
+  stop as serverJobsStop,
+  selectJobsByJobSequenceId,
+} from './lib/server-jobs';
```

Comment:
For consistency with others, I'd rather do one of three things:

- Continue using `import * as serverJobs ...`
- Rename `init`/`stop` to `initServerJobs`/`stopServerJobs`

I'd lean towards the former.

Reply:
Done in d85a7294b749155a6306497bec9b1ac2e446a068.